### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by TemesgenA**
## Featured Implemented
- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Added styles to improve the look and feel.
- For every page, a route is created

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nisi facere porro voluptatum corporis maxime, dicta dolorem eius exercitationem iste fuga autem esse earum vel libero corrupti debitis quibusdam repellendus similique, quidem, temporibus sequi quis sapiente hic molestiae. Sint similique tempora suscipit itaque! Quam ut reprehenderit minus a error sapiente porro.</p>
+      <p>
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quis suscipit necessitatibus culpa rem, temporibus sed cumque eum aliquam asperiores ipsum ipsa voluptatum iure qui id, totam at officia iste assumenda vitae ad hic sunt exercitationem. Dolorum unde obcaecati hic quae ex temporibus atque itaque voluptates eveniet iusto amet ipsa exercitationem cumque reiciendis quod ut fugit repudiandae assumenda similique, minima voluptate asperiores! Fuga minima laudantium odit maiores corporis molestias doloribus tenetur necessitatibus, doloremque saepe magnam cupiditate praesentium rem, placeat unde magni! Inventore itaque autem unde in praesentium placeat incidunt eos iusto, perspiciatis quidem vitae cupiditate cumque aut quibusdam similique soluta nihil.
+      </p>
+    </div>
```
Comment (mwanawabangona) [inline src/components/Home.js:12]: - [ ] `required`
- [ ] great job implementing the full site 👏 
- [ ] however there is need for adjusting in accordance to the template.
- [ ] please move the welcome message as well as the paragraph to the just as the wireframe. 🥂 
- [ ] The layout should match the wireframe please see screenshots below and marked areas.

`your wireframe`
image

`original wireframe`
image

File: src/components/Calculator.jsx [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
-    );
-  }
-}
+  const clickHandler = (event) => {
+    setState({ ...setState, ...calculate(state, event) });
+  };
+
+  return (
+    <div className="calculator">
+      <Display
```
Comment (mwanawabangona) [inline src/components/Calculator.jsx]: - [ ] `required`
- [ ] great job implementing the full site 👏 
- [ ] however there is need for adjusting in accordance to the template.
- [ ] please move the calculator to the `right` as well on left so as to match the wireframe. 🥂 
- [ ] The layout should match the wireframe please see screenshots below and marked areas.

`your template`
image

`original template`
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @TemesgenA ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — COMMENTED** [review]
Hi @TemesgenA , 

### Highlights ✨ 

- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**